pkg/antctl/antctl.go

```diff
@@ -38,6 +38,10 @@ type transformedVersionResponse struct {
 	AntctlVersion                     string `json:"antctlVersion" yaml:"antctlVersion"`
 }
 
+type transformedAgentInfoResponse struct {
```

Comment:
Question - why we need this wrapper struct?

Reply:
The struct member for response is called "TransformedResponse". I just keep it consistent.

Reply:
Ok to me if we can catch the 0.3.0 release, but I do hope we revisit all CLI design items after 0.3.0.
Ideally we should keep API/CLI compatible across some number of releases, but maybe not a big deal at this stage.